web-common/src/features/scheduled-reports/utils.ts

```diff
@@ -17,3 +67,93 @@ export function getDashboardNameFromReport(
     null
   );
 }
+
+export function extractNotification(
```

Comment:
"extract" seems like the wrong verb here

Reply:
We extract the slack and email related fields from then notifiers array.